velox/common/base/CMakeLists.txt

```diff
@@ -12,12 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-add_library(velox_exception Exceptions.cpp VeloxException.cpp Exceptions.h)
-target_link_libraries(
-  velox_exception PUBLIC velox_flag_definitions velox_process Folly::folly
-                         fmt::fmt gflags::gflags glog::glog)
+velox_add_library(velox_exception Exceptions.cpp VeloxException.cpp
+                  Exceptions.h)
+velox_link_libraries(
+  velox_exception
+  PUBLIC
+  velox_flag_definitions
```

Comment:
Not sure what indent style is used in velox CMake, but generally it is easier to read with indentation after the `PUBLIC` and `PRIVATE` keywords, if autoformatting with `cmake-format` you might need to tell it about the custom functions. Not critical, but the new indent appears less readable to me.

Reply:
Yeah good idea! I have #6861 open anyway so I'll add a cmake-format config file!